spring-data-jdbc/src/test/java/org/springframework/data/jdbc/repository/JdbcRepositoryIdGenerationIntegrationTests.java

```diff
@@ -107,15 +131,126 @@ void manuallyGeneratedIdForSaveAll() {
 		assertThat(immutableWithManualIdEntityRepository.findAll()).hasSize(2);
 	}
 
+	@Test // DATAJDBC-2003
+    @Sql(statements = "INSERT INTO EntityWithSeq(id, name) VALUES(1, 'initial value');")
```

Comment:
Something seems off with the formatting.

Reply:
Thanks! Reformatted